README.md

```diff
@@ -1 +1,16 @@
-# homebrew-butterfly
\ No newline at end of file
+# Homebrew Butterfly
+
+A centralized repository for Butterfly related brews.
+
+## Requirements
+
+* [Homebrew](https://github.com/Homebrew/brew)
+* Java 7+
+
+## Installation
+
+Run the following in your command-line:
+
+```sh
+$ brew install paypal/butterfly/butterfly
```

Comment:
Was not this supposed to be `brew install paypal/butterfly`?

Reply:
Sorry, not sure. Think we either have to have two commands - 'brew tap paypal/butterfly' then 'brew install paypal/butterfly' or can use single command but have to specify exact formula.